src/renderer/entities/transaction/lib/transactionService.ts

```diff
@@ -347,6 +348,12 @@ async function getTxWeight(
   return getExtrinsicWeight(extrinsic, options);
 }
 
+function getTxBytesLength(transaction: Transaction, api: ApiPromise): number {
```

Comment:
I think it's time to stop using `getExtrinsic` map. It always add more coupling and make it harder to decompose work with transactions later on